site_python_floripa/pages/contato/contato.md

```diff
@@ -0,0 +1,31 @@
+Aqui você encontra as diferentes redes sociais da Python Floripa, para se comunicar conosco e ficar por dentro de todas as novidades da nossa comunidade!
+
+- ### [Telegram](https://telegram.me/pythonfloripa)
+
+    A rede social mais ativa da comunidade! Participe para saber dos próximos eventos, ver anúncios de vagas para desenvolvedores(as) na região de Florianópolis, tirar (ou sanar) dúvidas referentes a nossa linguagem de programação favorita e muito mais!
```

Comment:
[IMO] Acho o termo "pessoa desenvolvedora" melhor do que "desenvolvedores(as)"

Reply:
vou alterar, também acho melhor. Dúvida: o que seria o IMO?

Reply:
IMO -> In My Opinion

Reply:
ahh lá no projeto da empresa, o time usa [nit] (de nitpicking), que com a ajuda do Google tradutor ficou como picuinha ehehehh